src/tests/likeDislikeComment.test.js

```diff
@@ -0,0 +1,255 @@
+        expect(res.body.message).to.deep.equal(
+          'You have successfully liked this comment.'
+        );
+        done();
+      });
+  });
+  it('Should return a message of sucess if a dislike is registered', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/comments/${commentId}/dislike`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'You have successfully disliked this comment.'
+        );
+        done();
+      });
+  });
+  it('Should return a message of success when a comment is liked after the user disliked it', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/comments/${commentId}/like`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'You have successfully liked this comment.'
+        );
+        done();
+      });
+  });
+  it('Should retrieve comments of the article with correct number of likes and dislikes', (done) => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${articleSlug}/comments`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res.body.comments[0].likes).to.equal(1);
+        expect(res.body.comments[0].dislikes).to.equal(0);
+        done();
+      });
+  });
+  it('Should return  a message sucess if a dislike is registered again', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/comments/${commentId}/dislike`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'You have successfully disliked this comment.'
+        );
+        done();
+      });
+  });
+  it('Should return a message sucess if a dislike again ', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/comments/${commentId}/dislike`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'You have successfully removed your dislike for this comment.'
+        );
+        done();
+      });
+  });
+  it('Should retrieve comments of the article with correct number of dislikes', (done) => {
```

Comment:
This message is misleading, looking at it, I expect to see the number of `dislikes`, but actually what you display here is the number of `likes`

Reply:
Thanks @Malaba6, done